src/k2/module/k23si/client/k23si_client.h

```diff
@@ -135,6 +136,7 @@ class K23SIClient {
     seastar::future<GetSchemaResult> getSchema(const String& collectionName, const String& schemaName, int64_t schemaVersion);
     seastar::future<CreateSchemaResult> createSchema(const String& collectionName, dto::Schema schema);
     seastar::future<CreateQueryResult> createQuery(const String& collectionName, const String& schemaName);
+    seastar::future<Duration> getTSOErrorbound();
```

Comment:
Let's make this and the TSO call just normal functions, no need for a future here since we know the error bound will be available if we have TSOs to talk to.